python/pyspark/ml/regression.py

```diff
@@ -1003,20 +1003,20 @@ class GBTRegressor(JavaEstimator, HasFeaturesCol, HasLabelCol, HasPredictionCol,
     def __init__(self, featuresCol="features", labelCol="label", predictionCol="prediction",
```

Comment:
Maybe add a versionChanged notice so people know the default lossType has changed?